homeassistant/bootstrap.py

```diff
@@ -150,28 +158,32 @@ def from_config_dict(config: Dict[str, Any],
     _LOGGER.info("Home Assistant core initialized")
 
     # stage 0, load logging components
-    for component in components:
-        if component in LOGGING_COMPONENT:
-            hass.async_create_task(
-                async_setup_component(hass, component, config))
-
-    await hass.async_block_till_done()
+    await asyncio.gather(*[
```

Comment:
If you're not gathering any results, use `asyncio.wait`

Reply:
Makes sense, although wouldn't we want to capture exceptions here too?